section/algorithms.html

```diff
@@ -374,9 +374,9 @@ <h4>Generate HTTP Signature</h4>
             <li>
                 Let <var>components</var> be an array of strings identifying which content (headers, etc.) to include in the signature.
                 <ol>
-                    <li>Let `components` default value be `<<"@method", "@target-uri">>`.</li>
+                    <li>Let |components| default value be `<<"@method", "@target-uri">>`.</li>
```

Comment:
initial value?

Also <<"item">> feels like a form of serialization of an array of strings, would be better to move that into item 6-8. Btw, it'd be nice to restructure those into a form of let canonicalData be ...